recipes/emsdk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,30 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.36.0"
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+    test_type = "build_requires"
+    generators = "cmake"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def build_requirements(self):
+        if self._settings_build.os == "Windows":
```

Comment:
```suggestion
        self.build_requires("cmake/3.18.2")
        if self._settings_build.os == "Windows":
```

Reply:
Is 3.18 required? I think we should modify the limit in the `CMakeLists.txt`, not add the build-requires..

Reply:
if 3.18 is really required by this version of Emscripten, we have no choice except using build_requires